tasks/source.ps1

```diff
@@ -0,0 +1,54 @@
+[CmdletBinding()]
+Param(
+  [string]$command = '',
+  [string]$name = '',
+  [string]$location = '',
+  [string]$user = '',
+  [string]$password = '',
+  [int]$priority = 0,
+  [boolean]$bypass_proxy = $false,
+  [boolean]$allow_self_service = $false,
+  [boolean]$admin_only = $false,
+  [string]$_task
+)
+
+if ($command -eq '') {
+  # $_task will be something like 'chocolatey::source_xxx'
+  # we want to extract the 'xxx' from the end as that's our command
+  $task_parts = $_task -split "chocolatey\:\:source_"
+  # parts[0] = ''
+  # parts[1] = 'xxx'
+  $command = $task_parts[1]
+}
+
+$cmd = @('choco', 'source', $command, '--limit-output', '--no-progress')
```

Comment:
Suggest renaming this variable from `cmd` to something like `ChocolateyInvocation` or `ChocolateyCommand` for clarity and to fit the general style of PowerShell scripting.